src/components/PeoplePage.jsx

```diff
@@ -0,0 +1,107 @@
+import React, { useEffect, useState } from 'react';
+import { useHistory, useLocation } from 'react-router-dom';
+import { getPeople } from '../api';
+import { PeopleTable } from './PeopleTabele';
+
+export function PeoplePage() {
+  const [people, setPeople] = useState([]);
+  const [isSortedBy, setIsSortedBy] = useState('asc');
+  const history = useHistory();
+  const location = useLocation();
+  const searchParams = new URLSearchParams(location.search);
+
+  useEffect(() => {
+    getPeople().then(setPeople);
+  }, []);
+
+  const query = searchParams.get('query') || '';
+  const handleChange = (event) => {
+    if (event.target.value) {
+      searchParams.set('query', event.target.value);
+    } else {
+      searchParams.delete('query');
+    }
+
+    history.push(`?${searchParams.toString()}`);
+  };
+
+  const sortBy = searchParams.get('sortBy') || '';
+
+  const handleSortBy = (event) => {
```

Comment:
```suggestion
  const handleSortBy = (sortBy) => {
```
Сюда передавай параметр(название колонки) по которому сортируешь